client/src/components/AuthModal.js

```diff
@@ -36,17 +39,25 @@ export const AuthModal = () => {
       const data = await request("/api/auth/login", "POST", { ...form });
       auth.login(data.token, data.userId);
       setModalisOpen(false);
+      window.location.reload();
     } catch (e) {}
   };
 
+  const logoutHandler = (event) => {
+    event.preventDefault();
+    auth.logout();
+    history.push("/");
+    window.location.reload();
```

Comment:
What an additional page reload is for?

Reply:
As this is a single page app, when surfing the pages it does not reload. 
The goal was when logged in to show the "My cabinet" page, otherwise hide. 
Unfortunately, this page appears only after reload.
For this reason I put window.location.reload();
if there is another way to solve the problem, please advise.